classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def enter_student(student)
+    @students.push(student)
+    student.classroom = self # make this student part of this classroom
+  end
```

Comment:
- [ ] When implementing the `enter_student` method, it's a good practice to ensure that the student is not already included in the classroom before adding them. You can achieve this by using a conditional statement. Before adding the student, check if the `@students` array already includes the student. If not, then proceed to add the student to the classroom and set their `classroom` attribute to the current classroom instance. This approach helps to avoid duplicate entries and ensures that each student is added to the classroom only once.

```suggestion
  def enter_student(student)
    @students.push(student) unless students.includes(self)?
    student.classroom = self # make this student part of this classroom
  end
```